tests/test_csc.py

```diff
@@ -329,213 +329,136 @@ async def test_camera_cable_wrap_following(self):
-            self.assertFalse(mock_azimuth.tracking_enabled)
-            self.assertFalse(mock_elevation.tracking_enabled)
+            assert not ccw_device.power_on
+            assert not ccw_device.enabled
 
-            tai = salobj.current_tai()
-            initial_azimuth = mock_azimuth.actuator.path.at(tai).position
-            initial_elevation = mock_elevation.actuator.path.at(tai).position
+            async with salobj.Controller(name="MTRotator") as rotator:
+                await self.remote.cmd_enable.start(timeout=STD_TIMEOUT)
+                assert ccw_device.power_on
+                assert ccw_device.enabled
 
-            # Check that tracking is rejected if not enabled
-            kwargs = self.make_track_target_kwargs(
-                azimuth=initial_azimuth, elevation=initial_elevation, taiTime=tai
-            )
-            with salobj.assertRaisesAckError():
-                await self.remote.cmd_trackTarget.set_start(
-                    **kwargs, timeout=STD_TIMEOUT
+                await self.assert_next_sample(
+                    self.remote.evt_cameraCableWrapFollowing, enabled=True
                 )
+                assert ccw_device.tracking_enabled
+                self.mock_controller.set_command_queue(maxsize=0)
+
+                assert self.mock_controller.command_queue.empty()
 
-            # Enable tracking and check mock axis controllers
-            await self.remote.cmd_startTracking.start(timeout=STD_TIMEOUT)
-            self.assertTrue(mock_azimuth.tracking_enabled)
-            self.assertTrue(mock_elevation.tracking_enabled)
-
-            # Slew and track until both axes are in position.
-            t0 = time.monotonic()
-            estimated_slew_time = 2  # seconds
-            tracking_task = asyncio.create_task(
-                self.track_target_loop(
-                    azimuth=initial_azimuth + 3,
-                    elevation=initial_elevation + 1,
-                    azimuth_velocity=0.003,
-                    elevation_velocity=0.003,
+                # Test regular tracking mode. CCW and MTRotator start in sync.
+                ccw_limits = mtmount.LimitsDict[mtmount.DeviceId.CAMERA_CABLE_WRAP]
+                # seconds until the CCW demand hits the position limit;
+                # make it large enough that we get a few target events
+                # before we reach the position limit.
+                trunc_time = 0.5
+                if do_max_limit:
+                    position_limit = ccw_limits.max_position
+                    velocity_limit = ccw_limits.max_velocity
+                else:
+                    position_limit = ccw_limits.min_position
+                    velocity_limit = -ccw_limits.max_velocity
+                velocity = velocity_limit * 1.1
+                position0 = position_limit - (velocity * trunc_time)
+                print(
+                    f"Position limit={position_limit}, velocity limit={velocity_limit}"
                 )
```

Comment:
Is this print useful for the unit test or a remainder of debugging the code?

Reply:
Good question. If something goes wrong it allows me to more quickly diagnose what the problem was. So I consider it part of the test.